### PR TITLE
Myfeature

### PR DESCRIPTION
**Pull request by baqar-abbas**
- Create a Pull request 
- Saved all the files here. Incorporated the suggested changes recommended by the code reviewer.
-  updated CSS file for managing spaces and indentations.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+h1 {
+ color: blue; 
+}
```
Comment (abdulhamiid) [inline styles.css]: image

Please kindly fix the Linters / Stylelint error, all linter results should be green ✔️. AKA: Linter should report NO errors.

To view the report in your terminal, Run  `npx stylelint "**/*.{css,scss}"`.  then you can solve the error 

Or automatically fix by using `npx stylelint "**/*.{css,scss} --fix"`

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @baqar-abbas 👋 , 
Alt Text
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️

- [ ] Please kindly customize all entries of your project **README file**, The readme file is used to explain our project and how we can install or use it for other developers, so let's make it more descriptive as possible. 💪 

- [ ] Please kindly update your Pull request summary
image

  - Pull Request should have a descriptive title telling you what changes are inside.
      
   - Pull Request should have a summary describing introduced changes in more detail. please let's fix that. 🤗

- [ ] Please make sure to include a LICENSE to your project, at the moment this is not working. Microverse suggests you use the MIT one. Follow the [official docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to achieve this! 🤓

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -268,13 +234,13 @@ I would like to thank...
...
+- **[Did you understand the Git Hub Process Flow]**
 
-  - [Answer_1]
+  - [Yes and will improve as I gain more experience working with Git Hub]
 
-- **[Question_2]**
+- **[Do you understand Git Process Flow]**
 
-  - [Answer_2]
+  - [Yes and will improve as I gain more experience working with Git]
```
Comment (TuyishimireEric) [inline README.md:243]: - [ ]  suggested by the previous Reviewer, Kindly provide a valid link for your project's License we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.

**Review by TuyishimireEric — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :recycle: 
### Hi  @baqar-abbas     👋 👋 
#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 
✔️No Linter Error 👍

## Required Changes ♻️

- [ ] **Kindly as suggested by the previous reviewer,** It would be better to make your Pull Request title more descriptive. You may rename it to the performed action and this will make it more Professional 👍😎


**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ABDULALI3468 — APPROVED** [review]
# APPROVED 🥇 

Hi @baqar-abbas , 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by baqar-abbas** [comment]
Thanks for the valuable feedback I am merging the Pull Request after your Approval.
Regards, Baqar Abbas

**Pull request merged**